src/quacc/recipes/espresso/phonons.py

```diff
@@ -342,6 +350,19 @@ def grid_phonon_flow(
         This will reduce the amount of data produced by a factor of nblocks.
         If nblocks = 0, each job will contain all the representations for a
         single q-point.
+    run_relax
+        If True, structure relaxation is performed using [quacc.recipes.espresso.core.relax_job][]
+        before running parallelized phonon jobs.
+    copy_files
+        Source directory or directories to copy files from. If a `SourceDirectory` or a
+        list of `SourceDirectory` is provided, this interface will automatically guess
+        which files have to be copied over by looking at the binary and `input_data`.
+        If a dict is provided, the mode is manual, keys are source directories and values
+        are relative path to files or directories to copy. Glob patterns are supported.
```

Comment:
Similarly, this `copy_files` keyword argument does not apply to all jobs. In fact, this keyword argument might potentially cause some confusion for users. If the user runs `grid_phonon_flow` with `run_relax = True` but a user also specifies `copy_files`, then the files being copied to the runtime directory in `ph_init_job` may not be those associated with the relaxed structure used as input (i.e. the output from `pw_job`).

Any suggestions on how you think we should approach this?